product_variant_default_code/models/product.py

```diff
@@ -100,12 +76,32 @@ class ProductTemplate(models.Model):
         "attribute name",
     )
 
+    @api.depends(
+        "code_prefix",
+        "attribute_line_ids",
+        "attribute_line_ids.attribute_id.name",
+        "attribute_line_ids.attribute_id.code",
```

Comment:
no sure that we need to depend on  "attribute_line_ids.attribute_id.code" here

Reply:
Indeed, not necessary